tests/containers/worker.pm

```diff
@@ -0,0 +1,19 @@
+use Mojo::Base qw(openQAcoretest);
+use testapi;
+use utils;
+
+sub run {
+  my ($self) = @_;
+  my $volumes = "-v \"/root/data/factory:/data/factory\" -v \"/root/data/tests:/data/tests\" -v \"/root/openQA/container/webui/conf:/data/conf:ro\"";
+
+  assert_script_run("cat '127.0.0.1 openqa_webui' >>/etc/hosts");
+  assert_script_run("echo -e '[openqa_webui]\nkey=1234567890ABCDEF\nsecret=1234567890ABCDEF' >/data/conf/client.conf");
+
+  assert_script_run("docker run --rm -d --network testing $volumes --name openqa_worker openqa_worker");
+
+  wait_for_container_log("openqa_worker", "Registering with openQA", "docker");
+  record_info("worker inizializated");
```

Comment:
typo in the string. Better just delete this line :)

Reply:
Done